### ISSUE TITLE
Press multiple movement keys

### ISSUE THREAD

**Issue by mjhorvath** [leozide/leocad#368]
Currently, the WASD keys control camera movement, but only one key can be pressed at a time. Could you make it so that I can press more than one key to get diagonal/simultaneous movement? Thank you.

**Comment by j6t** [comment]
This is not a game, and there is no game engine that would drive the keyboard directly. The application only responds to keypresses like any regular application. I don't think there is a way to implement this.

**Comment by mjhorvath** [comment]
Okay, I can do this in LDView and assumed it would be possible in LeoCAD too. Sorry.

**Comment by nathaneltitane** [comment]
![image](https://user-images.githubusercontent.com/5674757/152034027-c064e888-c78c-4915-91b9-ba1a7730a949.png)

**Comment by nathaneltitane** [comment]
ref #165 #443

**Comment by nathaneltitane** [comment]
> Okay, I can do this in LDView and assumed it would be possible in LeoCAD too. Sorry.

needs to be implemented

**Comment by j6t** [comment]
The image says:

> Arrow keys rotate the model. [...]

Do the arrow keys work like FPS controls, i.e., rotate continuously as long as the key is pressed, or does that just depend on auto-repeated key-presses? Do simultaneously pressed keys mix the rotation, or does the second key-press override the first one?

**Comment by nathaneltitane** [comment]
Yes, like fps... as long as the arrow is being pressed, the model rotates in the arrow's direction

**Comment by nathaneltitane** [comment]
> This is not a game, and there is no game engine that would drive the keyboard directly. The application only responds to keypresses like any regular application. I don't think there is a way to implement this.

ldview does it too btw...

**Comment by Lewiscowles1986** [comment]
This is not solidworks or ldview...

**Comment by nathaneltitane** [comment]
> This is not solidworks or ldview...

could you be any more condescending?

I was merely answering that it can be/has been done, or implemented...
